secboot/keymgr/keymgr_luks2.go

```diff
@@ -59,6 +59,17 @@ func getEncryptionKeyFromUserKeyring(dev string) ([]byte, error) {
 	return currKey, err
 }
 
+var keyslotFull = regexp.MustCompile(`^.* cryptsetup failed with: Key slot [0-9]+ is full, please select another one\.$`)
```

Comment:
This is an external error right so we can't match on error type? Isn't the match too precise, should we relax it a little bit?

Reply:
yeah, unfortunately there's no specific error coming from cryptsetup, so we have to inspect the error message which contains cryptsetup's stderr